parity/json_diff.py

```diff
@@ -0,0 +1,197 @@
+from copy import deepcopy
```

Comment:
Code to generate **algod** v. **indexer** parity reports. This could be converted into a circle test that:
1. pulls latest `go-algorand` submodule
2. runs the report
3. asserts that the results are as expected. In particular:
  * all _essential_ features of **algod** have their counterparts in **indexer** (think of `min-balance`)
  * terminology and responses remain consistent
  * etc.